iOS/Flows/Profile/ProfileViewController.swift

```diff
@@ -175,78 +163,89 @@ class ProfileViewController: DiffableCollectionViewController<UserConversationsD
     override func retrieveDataForSnapshot() async -> [UserConversationsDataSource.SectionType : [UserConversationsDataSource.ItemType]] {
         return [:]
     }
-    
-    private func loadArchive() {
-        Task { [weak self] in
-            guard let `self` = self, let user = self.person as? User else { return }
+
+    // MARK: - Conversation Loading
+
+    /// The currently running task that is loading conversations.
+    private var loadConversationsTask: Task<Void, Never>?
```

Comment:
Previously if you tapped quickly between tabs the collection view would get confused. Now we can cancel previous load calls to prevent that.